templates/person/fulltext.html.twig

```diff
@@ -1,14 +1,7 @@
 {% extends 'base.html.twig' %}
```

Comment:
It has been around since 2020, but untouched since: https://github.com/sfu-dhil/wphp/commits/main/templates/person/fulltext.html.twig, so I imagine it can be deleted?

Reply:
I think so. It was probably missed removing one of the many failed search experiments.